changelog/unreleased/3435

```diff
@@ -0,0 +1,7 @@
+Bugfix: Scoped Storage Wizard Text
```

Comment:
From my point of view, we don't need the calens item here. This is just an update about a text that was not released in a stable channel. So it's not needed to add a calens item for it.

Reply:
Okay, perfect!